test/e2e-ocl/onclusterlayering_test.go

```diff
@@ -238,30 +242,35 @@ func TestMissingImageIsRebuilt(t *testing.T) {
 	waitForMOSCToGetNewPullspec(ctx, t, cs, moscName, thirdImagePullspec)
 
 	// Apply the MC again
-	t.Logf("Applying MachineConfig %q", firstMC.Name)
+	t.Logf("Re‐applying the same MachineConfig %q to confirm no new build for image2", testMC.Name)
 	secondMC, err := cs.MachineConfigs().Create(ctx, testMC, metav1.CreateOptions{})
 	require.NoError(t, err)
 	kubeassert.MachineConfigExists(secondMC)
 	t.Logf("Created MachineConfig %q", secondMC.Name)
-	t.Logf("Reusing previous MachineOSBuild %q that has produced image: %s", secondFinishedBuild.Name, secondImagePullspec)
-	waitForMOSCToGetNewPullspec(ctx, t, cs, moscName, secondImagePullspec)
 
-	// Delete the third MOSB and the image should be deleted also
-	t.Logf("Deleting MachineOSBuild %q", thirdMOSB.Name)
+	// waitForMOSCToGetNewPullspec(ctx, t, cs, moscName, secondImagePullspec)
```

Comment:
thought (non-blocking): If this isn't being used, it's better to delete the line and let Git worry about what was once here.